### DISCUSSION TITLE
Lesson 10: Can not repay out borrows/loans AAVE UI

### DISCUSSION THREAD

**Discussion by sevamove** [smartcontractkit/full-blockchain-solidity-course-py#244] [Q&A] [answered]
It seems that Aave has changed their URL from `https://testnet.aave.com` to `https://staging.aave.com` and some of their functionality was lost.

For illustration: I can not anymore repay out ([video](https://youtu.be/M576WGiDBdQ?t=31516)) the smallest amount of DAI (0.296 DAI) borrows/loans with my current collateral because there is simply no button after I click repay. I have enough Ether to be able to do that.

Does anyone get this kind of problem?

**Comment by cromewar** [comment, marked as answer]
Yeah this scenario is probably going to happen often, blockchain technologies and platforms changes really fast... 

If I remember well I faced the same problem and it was because metamask didn't pump up the confirmation, I suggest go to your dashboard and try again after cleaning all your browser cookies ans history., may be that should work.

**Reply by PatrickAlphaC** [reply]
Dang.... Aave why you do this.

Thanks for the answer cromewar. Looks like that part of the video might be defunc. I'll update the issues docs.
